### DISCUSSION TITLE
OpenRA working?

### DISCUSSION THREAD

**Discussion by DocMAX** [ptitSeb/box64#1213] [General]
I try to run OpenRA (https://www.openra.net/) on my Mali G72 phone (VirGL). Tried differend wine and libGL versions. No chance. Anyone here got it working with similar hardware? OpenRA shouldn't be too demanding on hardware right?
I would love to have OpenRA portable right in my pocket.

Last lines i get with wine-ge-custom 8.0 and wine 8.07 staging:
```
No headers available
failed to get fd
Unable to get a valid fd
No headers available
failed to get fd
Unable to get a valid fd
No headers available
failed to get fd
Unable to get a valid fd
0158:err:d3d:match_fbo_tex_update FBO status 0
```

**Comment by ptitSeb** [comment]
That looks like a Graphic Driver Issue. Can you simple OpenGL stuff?

**Reply by DocMAX** [reply]
Other OpenGL apps work (not all of course). GPU Caps Viewer works, every demo up to GL 4.3.

**Reply by DocMAX** [reply]
@ptitSeb Can you help make this game work please?

**Reply by ptitSeb** [reply]
I don't think I can help here. I still think It's a graphic driver issue. Doesn't openra can be used with OpenGL? Because it seems to want to use dx9.

**Reply by DocMAX** [reply]
It's working!!!! :-) I had to set MESA_GL_VERSION_OVERRIDE to 3.2!

**Reply by ptitSeb** [reply]
That was simple. The right OpenGL version and the game is happy!

**Reply by DocMAX** [reply]
Maybe someone can make a guide what steps to try step by step if a game is not starting in the first place.
